docs/source/_json/types.resp

```diff
@@ -3,63 +3,68 @@ Content-Type: application/json
 
 [
   {
-    "@id": "http://localhost:55001/plone/@types/Collection", 
```

Comment:
@jaroel you really have to stop opening the generated _json files with your editor. :) This makes it hard to review your pull requests.